### DISCUSSION TITLE
Feedback on repo description editing

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#62937] [Mobile]
I am really glad to see that GitHub mobile team have recently added functionality to edit repo description on mobile, this is really helpful.

But I am here to implore you to make more improvements to the application, please provide functionality to rename a repo, functionality for a repo to show tags, add and remove tags from a repo, rename branches, and improve the code editing interface.

**Comment by mehulrajdev** [comment]
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

Correct ?
Mark as answer

**Reply by NitinIsFit** [reply]
Not correct, there is still no functionality to rename a repo
